site/zenodo_rdm/legacy/serializers/__init__.py

```diff
@@ -33,6 +33,27 @@ def unwrap_hits(self, data, many, **kwargs):
         return data.get("hits", {}).get("hits", [])
 
 
+class ZenodoListSchema(BaseListSchema):
+    """Legacy top-level array/list schema for JSON Serializer."""
+
+    links = fields.Method("get_links")
+    sortBy = fields.Method("get_sorting_option")
```

Comment:
I've checked and ``sortBy`` was not included in the legacy JSON, but since we'll be adding in ``BaseListSchema`` we'll need to set it to None here.

```suggestion
    sortBy = None
```